lib/schema-evolution-manager/apply_util.rb

```diff
@@ -10,6 +10,7 @@ def initialize(db, opts={})
 
       @db = Preconditions.assert_class(db, Db)
       @scripts = Scripts.new(@db, Scripts::SCRIPTS)
+      @pgpass_file = Preconditions.assert_class(pgpass_file, Tempfile)
```

Comment:
I think better is to make the argument of type string - the path to a file - rather than requiring a Tempfile